main.cpp

```diff
@@ -9,12 +9,22 @@ struct A{
 int main(){
 //  Dynamic allocation of struct A
     cout << "Dynamic allocation of struct A" << endl;
-    A *objA = (A*)malloc(sizeof(A));
```

Comment:
malloc() allocates memory and does not call the constructor. new() is more useful.